openscm/definitions/openscm_units.txt

```diff
@@ -0,0 +1,64 @@
+# syntax is either:
+# for variables which relate to existing variables in the registry
+# name = relation to unit already in registry = other names
+# for variables which should define a new base unit
+# name = [long_name] = other names
+
+a = 1 * year = annum = yr
+kt = 1000 * t
```

Comment:
```suggestion
h = hour
# since kt is used for "knot" in the defaults:
kt = 1000 * t
```

Reply:
Do we need the defaults like "knot" at all?